github/pull_request.go

```diff
@@ -0,0 +1,51 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/google/go-github/v64/github"
+)
+
+func GetPullRequestFiles(client *github.Client, ctx context.Context, o, r string, n int) ([]*github.CommitFile, *github.Response, error) {
+	files, resp, err := client.PullRequests.ListFiles(ctx, o, r, n, nil)
```

Comment:
I don't think we need this function we can just use this function directly, essentially all this function is doing is wrapping the error message

Reply:
The idea of the repo is that you don't have to repeat code in other scripts its all done here and tested here and is always on the latest package version that way only have to update one package across all repos